game.cpp

```diff
@@ -194,12 +217,17 @@ bool Game::RunFrame(double dt) {
     //m_pPlayerEntity->UpdateCamera(&m_FollowCamera);
 
     // Render stuff
-    IRender* renderer = GetRenderer();
 
     // ImGUI stuff goes into GL default FBO
 
     ImGui::ShowDemoWindow();
 
+    ImGui::Begin("Statistics");
+    ImGui::Text( "World Tri count: %d", m_World->StaticGeometryCount() );
+    ImGui::Text( "Brush Entity count: %d", m_World->GetModelPtrs().size() );
+    ImGui::Text( "Total Entity count: %d", EntityManager::Instance()->Entities().size() );
+    ImGui::End();
```

Comment:
we should think about adding a unified ui for informative stats to imgui that can be toggled via the console. with the new first person view it's quite tricky to move the windows of imgui.

Reply:
Agreed. It is annoying when the cursor cannot be put on screen when in FPS mode. I am going to make a fix for that.

I would let @Cameramorphic come up with an idea on building an ImGUI stats UI that can be toggled via console.